bucket/chromium.json

```
@@ -1,37 +1,49 @@
 {
-    "version": "75.0.3770.142-r652427",
-    "description": "An open-source browser project that aims to build a safer, faster, and more stable way for all users to experience the web.",
+    "##": "Check chromium.woolyss.com for different versions of Chromium releases.",
+    "version": "76.0.3809.100-r665002",
+    "description": "Open-source browser project that aims to build a safer, faster, and more stable way for all users to experience the web.",
     "license": "BSD-3-Clause",
     "homepage": "https://www.chromium.org",
-    "checkver": {
-        "url": "https://chromium.woolyss.com/api/v3/?os=windows&bit=64&type=stable-codecs-sync&out=string",
-        "re": "v([\\d.]+-r(?:\\d+))-win64"
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v76.0.3809.100-r665002-Win64/Chromium-76.0.3809.100-x64.7z",
+            "hash": "sha1:67fda51228441ec59bdbfa4de046f4ea4f4d551b",
+            "extract_dir": "Chromium-76.0.3809.100-x64"
+        },
+        "32bit": {
+            "url": "https://github.com/macchrome/winchrome/releases/download/v76.0.3809.100-r665002-Win64/Chromium-76.0.3809.100-x86.7z",
+            "hash": "sha1:b3a9c7bfaf220ba2f0868047e1043b74246a9c96",
+            "extract_dir": "Chromium-76.0.3809.100-x86"
+        }
     },
     "bin": "chrome.exe",
```

Is it convient to use just `chrome`? This will conflict with googlechrome

```suggestion
    "bin": [
        "chrome.exe",
        [
            "chrome.exe",
            "chromium"
        [
    ],
```

When there are multiple version of chrome sideloaded it still will be overriden, but new bin will be available without needing to reset all the time.

---

Agree. It will be more convenient this way.
Thanks for the advice.